src/components/tool/guide/EresearchJob/EresearchInstructions.js

```diff
@@ -12,31 +12,50 @@ import React from "react";
 import InstructionInput from "../Instructions/InstructionInput";
 
 const TEXT_PATTERN = /^[a-z0-9]*$/;
+const JOB_NAME_PATTERN = /^(?=.*[a-z0-9-])[a-z0-9-]+$/;
 
 
 export function EresearchInstructions({config}) {
     const [username, setUsername] = useLocalStorageState("qutUsername", "qutusername");
+    const [batchJobName, setBatchJobName] = useLocalStorageState("qutBatchJobName", "batch-job-001");
 
     return <Box>
         {config.service === "Lyra" &&
-            <InstructionInput
-                label="QUT Username"
-                placeholder="username"
-                value={username}
-                onChange={setUsername}
-                pattern={TEXT_PATTERN}
-                helperText="Optionally enter your QUT username so that it can be pre-populated in the commands below."
-                errorText="Username must only contain lowercase letters and numbers"
+            <>
+                <InstructionInput
+                    label="QUT Username"
+                    placeholder="username"
+                    value={username}
+                    onChange={setUsername}
+                    pattern={TEXT_PATTERN}
+                    helperText="Optionally enter your QUT username so that it can be pre-populated in the commands below."
+                    errorText="Username must only contain lowercase letters and numbers"
+                />
+                {config.jobType === "Batch" &&
+                    <InstructionInput
+                        label="Batch Job Name"
+                        placeholder="batch-job-001"
```

Comment:
I think we shouldn't specify a placeholder. If it's needed, it shouldn't have numbers in it as that would be unusual/confusing for batch jobs

Reply:
I think it is needed to have a placeholder so that a user would have an idea what is it and how should it look like, but I agree that we should remove the numbers for now. I'll update the regex pattern